_ml-commons-plugin/api/model-apis/batch-predict.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title:  Batch inference
+parent: Model APIs
+grand_parent: ML Commons APIs
+nav_order: 20
+---
+
+# Batch inference
+
+ML Commons can predict large datasets in an offline asynchronous mode with your remote model deployed in external model servers. To use the Batch_Predict API, the `model_id` for a remote model is required. This new API is released as an experimental feature in the OpenSearch version 2.16, and only SageMaker, Cohere, and OpenAI are verified as the external servers that support this features.
+
+For information about user access for this API, see [Model access control considerations]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/index/#model-access-control-considerations).
+
+
+For information about connectors and remote models, see [Connecting to externally hosted models]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/index/). For more details of the connector blurprints for batch predict, see [GitHub docs](https://github.com/opensearch-project/ml-commons/tree/main/docs/remote_inference_blueprints)
```

Comment:
I don't think this is a good customer experience. We are asking customer to check pool of blue prints which might not related to them. Can we point them to any specific blueprints or may be we can create another folder for `batch_prediction` and point them there?